api/Document.json

```diff
@@ -5212,7 +5222,7 @@
                 "version_added": "79"
               },
               {
-                "version_added": "≤79",
+                "version_added": "12",
```

Comment:
Note that `document.fullscreen` was added to the spec as Firefox was trying to ship the unprefixed API and ran into some compat issues. It didn't exist at the time of Edge 12, so it's omission here is correct.